ev3/legoev3_motor.c

```diff
@@ -385,24 +358,16 @@ static void legoev3_motor_set_power(struct legoev3_motor_data *ev3_tm, int power
  */
 static void legoev3_motor_reset(struct legoev3_motor_data *ev3_tm)
 {
-	const struct legoev3_motor_info *info = &ev3_tm->info->legoev3_info;
-
 	memset(ev3_tm->tacho_samples, 0, sizeof(unsigned) * TACHO_SAMPLES);
 
 	ev3_tm->tacho_samples_head	= 0;
 	ev3_tm->got_new_sample		= false;
-	ev3_tm->num_samples		= info->samples_for_speed[SPEED_BELOW_40];
+	ev3_tm->num_samples		= ev3_tm->info->legoev3_info.samples_for_speed[SPEED_BELOW_40];
```

Comment:
Wrap lines to 80 characters. This is why `const struct legoev3_motor_info *info = &ev3_tm->info->legoev3_info;` was there. For readability.